third_party/closure/goog/caja/string/html/htmlparser.js

```diff
@@ -209,7 +209,6 @@ goog.string.html.HtmlParser.Elements = {
 /**
  * Regular expression that matches &s.
  * @type {RegExp}
- * @private
```

Comment:
Are these used anywhere besides this file and htmlsanitizer.js? If not we can probably make them `@package`.